pkg/config/config.go

```diff
@@ -83,6 +83,7 @@ type DefaultConfig struct {
 type CoeConfig struct {
 	Cluster          string `ini:"cluster"`
 	EnableVPCNetwork bool   `ini:"enable_vpc_network"`
+	EnableWebhook    bool   `ini:"enable_webhook"`
```

Comment:
do we need to set a new option under coe?

Reply:
Removed.